src/commands.cpp

```diff
@@ -5162,12 +5162,16 @@ void CommandSetKeyPair::procresult()
 
     client->json.storeobject();
 
-    client->key.ecb_decrypt(privkBuffer, len);
-    client->mPrivKey.resize(AsymmCipher::MAXKEYLENGTH * 2);
-    client->mPrivKey.resize(Base64::btoa(privkBuffer, len, (char *)client->mPrivKey.data()));
+    if (privkBuffer)
```

Comment:
I use it because the memory is allocated in constructor but it's remove in procresult, I know that it's not allow call twice procresult but just in case. Use a unique_ptr is a better option as you say in the other comment